pkg/nixbase32/nixbase32.go

```diff
@@ -8,6 +8,42 @@ import (
 // Alphabet contains the list of valid characters for the Nix base32 alphabet.
 const Alphabet = "0123456789abcdfghijklmnpqrsvwxyz"
 
+// ValidateString validates if a byte slice is valid nixbase32.
+func Validate(b []byte) error {
```

Comment:
The comment still stays `ValidateString`, but it's accepting bytes. Maybe we should call them `Validate{String,Bytes}`?

Reply:
Done.